pygmt/tests/test_datasets_earth_relief.py

```diff
@@ -52,8 +52,8 @@ def test_earth_relief_01d_gebco(data_source):
     assert data.gmt.registration == 0
     npt.assert_allclose(data.lat, np.arange(-90, 91, 1))
     npt.assert_allclose(data.lon, np.arange(-180, 181, 1))
-    npt.assert_allclose(data.min(), -8598)
-    npt.assert_allclose(data.max(), 5559.0)
+    npt.assert_allclose(data.min(), -8597.0, atol=1.0)
+    npt.assert_allclose(data.max(), 5559.0, atol=1.0)
```

Comment:
Same here for the GEBCO grids, is the 1.0 uncertainty value from some paper? Not seeing it at https://www.generic-mapping-tools.org/remote-datasets/earth-gebco.html.